src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityWidgetV2.java

```diff
@@ -325,18 +319,11 @@ private void configureActions() {
 
     this.actionMenu.setActionListener(
         Action.EDIT_ENTITYREF_COLLECTION_ITEMS,
-        action -> {
-          showDatasetItemsEditor();
-        }
+        (action, e) -> showDatasetItemsEditor()
       );
 
     // Edit data
-    this.actionMenu.setActionEnabled(Action.EDIT_TABLE_DATA, hasQueryableData);
```

Comment:
Looks like we used to only enable this command if there was queryable data (and set the tooltip text to explain why it might not be enabled).  Now it's only visible if it has queryable data.

Reply:
Thanks for catching this, I must have blindly removed when I removed the `setEditTableDataTooltipText` method.